core/iwasm/interpreter/wasm_opcode.h

```diff
@@ -791,14 +791,15 @@ typedef enum WASMAtomicEXTOpcode {
 #define SET_GOTO_TABLE_ELEM(opcode) [opcode] = HANDLE_OPCODE(opcode)
 
 #if (WASM_ENABLE_JIT != 0 || WASM_ENABLE_FAST_INTERP != 0) \
-    && WASM_ENABLE_SIMD != 0
+    && WASM_ENABLE_SIMD != 0 && WASM_ENABLE_SIMDE != 0
 #define SET_GOTO_TABLE_SIMD_PREFIX_ELEM() \
     SET_GOTO_TABLE_ELEM(WASM_OP_SIMD_PREFIX),
 #else
 #define SET_GOTO_TABLE_SIMD_PREFIX_ELEM()
 #endif
 
-#if (WASM_ENABLE_FAST_INTERP != 0) && WASM_ENABLE_SIMD != 0
+#if (WASM_ENABLE_FAST_INTERP != 0) && WASM_ENABLE_SIMD != 0 \
+    && WASM_ENABLE_SIMDE != 0
```

Comment:
After #4260, `WASM_ENABLE_SIMDE` represents fast-interpreter + SIMD support, so I guess  `#if WASM_ENABLE_SIMDE !=0 ` is sufficient here